Update customize-the-app-launcher.md (#7036)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/customize-the-app-launcher.md

1. In the admin center, go to the **Settings** > **Org Settings** > **Organization profile** tab.
    

> 2. On the **Organization profile** page, go to **Custom App launcher tiles**,  If you select the three dots next to your **Custom Tile** and Select **Edit custom tile**.

Reviewer: ```suggestion
2. On the **Organization profile** page, go to **Custom app launcher tiles**. Select the three dots next to your custom tile and then select **Edit custom tile**.
```
